### ISSUE TITLE
Nested namespaces

### ISSUE THREAD

**Issue by Night-walker** [daokoder/dao#361]
test1.dao:
```ruby
namespace one {x}

var x = 1
```
test2.dao:
```ruby
namespace two {y}

load test1

var y = 2
```
Let's try it out:
```ruby
(dao) load test2
= none
(dao) one
= namespace_19_0x8e4ea0
(dao) one.x
= 1
```
This is not how namespaces should function. Not how I expect them to, at least. If namespace `one` is not specified within `namespace two {...}`, it should not be exported further. And if exported, it should become `two.one`, respectively. Otherwise we cannot have 'service' namespaces intended for internal use (and which do not pollute the global namespace).

**Comment by dumblob** [comment]
I agree, that `one` and all it's members shouldn't be visible if loaded the way shown above (i.e. transitively), but I don't agree with the implicit `two.one` behavior. To be able to use `two.one` one needs to explicitly export the symbol `one` using e.g. `namespace two {y, one}` and it should stay this way.

**Comment by Night-walker** [comment]
> To be able to use two.one one needs to explicitly export the symbol one using e.g. namespace two {y, one} and it should stay this way.

I meant exactly this. No implicitness.
